### ISSUE TITLE
Interested on contribute.

### ISSUE THREAD

**Issue by andres-root** [mozillascience/training_for_libraries#1]
This project sounds great, can I contribute somehow? I'm a Python/Javascript developer but I'm better with backend stuff.

**Comment by auremoser** [comment]
yes please! sorry I've been so delayed in setting things up, we're prepping it as an open source project that aggregates a lot of coding resources for librarians. feel free to fork and pull request and i'll merge whatever you have to offer, or just enter links as issues and I'll add them in. I'd like to have it setup for MozFest in two weeks, trying to meet that deadline while doing other work stuff.

**Comment by andres-root** [comment]
ok, I can create a site with Python/Django in one week, with a basic authentication system and some forms working, but any idea of what to use in the frontend?

**Comment by auremoser** [comment]
JS? that's my jam, but i was actually planning on just making a [Gitbook](https://www.gitbook.io/) of curriculum tutorials and options based on the topics outlined in the readme...and attach it to the repo via a gh-pages branch. What were you thinking?

**Comment by andres-root** [comment]
I was thinking on build a platform for that, but with Gitbook it would be faster
